### DISCUSSION TITLE
Github doesn't verify me for their student programme.

### DISCUSSION THREAD

**Discussion by nerfelitewar** [community/community#40904] [GitHub Education]
### Select Topic Area

Show & Tell

### Body

Hi I am a high school student, and I wanted to use benefits of GitHub student program and  I  applied for that but I got rejected it said- Cant verify you properly so I retried and again it said the same thing. 
Also the website isn't working for me too, every time I try it says 404 Error. 

I tried to contact GitHub they said "Try again with school ID" but still it doesn't help me.

<h4>If you have any solution then please let me know. I'll be happy!</h4>

**Comment by ossanna16** [comment]
Hi there @nerfelitewar and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Reply by nerfelitewar** [reply]
Thanks for replying! 
I did my simple introduction on your official introduction thread :D 
And I hope my github student programme issue gets fixed ASAP 😓 as I've been trying hard to get that work but its not working for me which is awkward 😅...

**Comment by samus-aran** [comment]
Hi @nerfelitewar 👋 , we're happy you're here! You are more likely to get a useful response if you are explicit about what your reasons rejection are (a screenshot may help) and any additional info you think is relevant. Giving a few more details might help someone give you a nudge in the right direction. 👍

We're sorry to hear that you are receiving 500 errors when applying. Are you able to try again and take additional screenshots when the page fails to continue?

**Reply by nerfelitewar** [reply]
Well, it was happening before but when I checked it today it was fine. 
I tried to resubmit it as previous one got rejected, hopefully this time it gets accepted. 
I'm excited.

I will tell you if any issues come up with that again, thanks for your support :D
